UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<blockquote>below the name 'FLUX_K' it indicates it is a magnitude!</blockquote><p>below the name 'FLUX_K' the unit indicates it is a magnitude!</p><p> </p><p>Rather than "problematic" values, are they "missing" values? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>